test/e2e/util/util.go

```diff
@@ -39,6 +41,14 @@ const (
 	PodStartTimeout = 5 * time.Minute
 )
 
+var EnableYurtAutonomy = flag.Bool("enable-yurt-autonomy", false, "switch of yurt node autonomy. If set to true, yurt node autonomy test can be run normally")
+var RegionID = flag.String("region-id", "", "aliyun region id for ailunyun:ecs/ens")
+var NodeType = flag.String("node-type", "minikube", "node type such as ailunyun:ecs/ens, minikube and user_self")
+var AccessKeyID = flag.String("access-key-id", "", "aliyun AccessKeyId  for ailunyun:ecs/ens")
+var AccessKeySecret = flag.String("access-key-secret", "", "aliyun AccessKeySecret  for ailunyun:ecs/ens")
+var Kubeconfig = flag.String("kubeconfig", "", "kubeconfig file path for OpenYurt cluster")
+var ReportDir = flag.String("report-dir", "", "Path to the directory where the JUnit XML reports should be saved. Default is empty, which doesn't generate these reports.")
```

Comment:
Since we will not build any binary for e2e test. These option settings seem to be useless.